core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -325,5 +325,6 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
     {{ include_js_file('components/SharingLinksDirective.js') }}
     {{ include_js_file('dashboard/Dashboard.js') }}
     {{ include_js_file('domain/dashboard/DashboardBackendApiService.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
```

Comment:
@gvishal I thought you were dropping this import from this and all other pages where it isn't needed?